### DISCUSSION TITLE
Mealie as appliance OS image for Raspberry Pi

### DISCUSSION THREAD

**Discussion by kbsmidt** [mealie-recipes/mealie#2113] [Feature Request]
I would love to see Mealie delivered as an appliance OS image for the Raspberry Pi as i'll make it a lot easier for people to be on-boarded.

With a hardened OS with auto-installing security patches and auto-backing up the database to an external usb memory stick I'm willing to pay a license or a support fee to keep the project on-going.

**Comment by michael-genson** [comment]
Currently there are no plans to support any deployment methods other than Docker. We're happy to accept PRs for community guides that go in-depth on how to run Mealie elsewhere.
